### PR TITLE
Add kH display, open/close status and some other small fixes

### PR DESCRIPTION
**Pull request by aprilzhang**

### CODE REVIEW

File: src/lib/components/WellCompletions/utils/sort.ts [hunk 1]
```diff
@@ -12,9 +12,9 @@ const createSortKeyFunction = (
             return (well) => well.name;
         case SORT_BY_STRATIGRAPHY_DEPTH:
             return (well) =>
-                well.zoneIndices.find(
-                    (_, index) => well.completions[index] > 0
-                );
+                well.completions.find(
+                    (_, index) => well.completions[index]!.open > 0
```
Comment (alexandruandrei91) [inline src/lib/components/WellCompletions/utils/sort.ts:16]: This triggers an `Forbidden non-null assertion` warning.

**Review by alexandruandrei91 — COMMENTED** [review]
It looks good, but I have some questions about the behavior, so I'll give you a call on teams.

File: src/lib/components/WellCompletions/utils/sort.ts (hunk 1 above)
Comment (aprilzhang) [reply src/lib/components/WellCompletions/utils/sort.ts:16]: Yes I saw that too. Not sure what's the best way to fix the warning as well.completions[index] is obviously not undefined.

Comment (aprilzhang) [reply src/lib/components/WellCompletions/utils/sort.ts:17]: Added a comment to disable the warning for that line

**Review by alexandruandrei91 — APPROVED** [review]
👍

**Pull request merged**